cmd/earlystopping/medianstop/v1beta1/Dockerfile

```diff
@@ -7,7 +7,10 @@ ENV PYTHONPATH ${TARGET_DIR}:${TARGET_DIR}/pkg/apis/manager/v1beta1/python
 
 RUN if [ "${TARGETARCH}" = "ppc64le" ] || [ "${TARGETARCH}" = "arm64" ]; then \
     apt-get -y update && \
-    apt-get -y install gfortran libopenblas-dev liblapack-dev && \
+    apt-get -y install gfortran libopenblas-dev liblapack-dev libssl-dev pkg-config gcc g++ && \
```

Comment:
we shouldn't be installing packages like `libssl-dev pkg-config gcc g++` for arm64 if not required